.github/workflows/CI-windows.yml

```diff
@@ -90,18 +90,27 @@ jobs:
           windeployqt Build\gui || exit /b !errorlevel!
           del Build\gui\cppcheck-gui.ilk || exit /b !errorlevel!
           del Build\gui\cppcheck-gui.pdb || exit /b !errorlevel!
-          
-      - name: Build CLI debug configuration using MSBuild
+
+      - name: Configure MSBuild
```

Comment:
I don't understand what these changes in the yaml achieve for the colors?

Reply:
Windows build was broken because it doesn't regenerate the visual studio project. Furthermore, regenerating the project uses absolute paths so it can't be committed, so I just regenerate the project in the CI.